packages/cli/test/functional/test_site_special_tags/expected/markbind/js/initAppNode.page-vue-render.js

```diff
@@ -0,0 +1,7 @@
+
+      var body = document.querySelector('body');
```

Comment:
👀 not sure if this part is a good way to reduce fouc; it does remove the flash completely, but at the cost of a longer time to first-content paint.

this should be fixed with ssr as well

Reply:
Hmm, should we implement SSR in this PR as well? If not, do you think this implementation (at the cost of longer first-content paint time) is alright until we can implement the SSR? Personally I think it's better to have a longer time to first-content paint than FOUC. How should we approach this? :O

Reply:
> If not, do you think this implementation (at the cost of longer first-content paint time) is alright until we can implement the SSR?

That's subjective, but I think I would personally prefer having some content show up first instead of a blank white screen with a spinning wheel so I know my browser hasn't hanged. =P That said, first-content-paint (not tti) is one of the very common [metrics](https://web.dev/first-contentful-paint/).

But more importantly, no hurry to fix FOUC completely. Let's fix this in proper when implementing SSR, otherwise it would be removed then anyway =P

Reply:
Understood! I'll remove the script here and look to work on the SSR :-)